### PR TITLE
[EB-1432] showCustomerSettingsAsync implementation

### PR DESCRIPTION
**Pull request by vdanylov**

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,114 @@
+Arguments: 
+  /usr/local/bin/node /usr/local/Cellar/yarn/1.5.1_1/libexec/bin/yarn.js test -t should get error customer statusCode 401
```
[the remark this answers is not in the document]
Comment (vdanylov) [reply yarn-error.log]: yarn-error.log added to .gitignore

File: src/data/customer/Setting.js [hunk 2]
```diff
@@ -68,9 +71,35 @@ export const getCustomerSettingsAsync = async (token) => {
...
+    const CustomerSettingsShow = await axios({
+      method: 'GET',
+      url: `${endpoints.CUSTOMER_SETTINGS}/${customerId}/${
+        paramsString ? `?${paramsString}` : ''
+      }`,
+      headers: {Authorization: token},
     });
+    return {
+      ...CustomerSettingsShow,
+      data: camelize(CustomerSettingsShow.data),
```
Comment (vinay3206) [inline src/data/customer/Setting.js]: I think u just need to return camelize(CustomerSettingsShow.data)

File: src/data/customer/Setting.js [hunk 3]
```diff
@@ -68,9 +71,35 @@ export const getCustomerSettingsAsync = async (token) => {
...
+
+/** Retrieve Customer's Settings (reduced)
+ * @param {string} token
+ * @param {int} customerId
+ * @param {Object} payload {identityId, transactionGroupId}
+ * @return {Promise} settings object
+ */
+export const showCustomerSettingsAsync = async (
+  token,
+  customerId,
```
Comment (maartenhemmes) [inline src/data/customer/Setting.js]: I don't think customerId is required, as per documentation?

Comment (vdanylov) [reply src/data/customer/Setting.js]: @maartenhemmes we discussed it with Vinay and he sent me:

Comment (vinay3206) [reply src/data/customer/Setting.js]: @maartenhemmes API url is settings/{id}

id needs to be replaced by customerId

Comment (vdanylov) [reply src/data/customer/Setting.js]: @maartenhemmes it's done, I've replace id with customerId. Is it right? Or You mean that syntax should be id, but by logic it should be customerId?

Comment (maartenhemmes) [reply src/data/customer/Setting.js]: @vdanylov it's fine, if you want to optimise you could do the same as on other functions:

- make the {0} part of the url on Endpoint.js:   CUSTOMER_DRIVERS: `${urlPrefix}/customers/{0}/drivers`,

- Add a replace function:       url: endpoints.CUSTOMER_DRIVERS.replace('{0}', customerId),

This way it's consistent with the other functions.

File: src/data/customer/Setting.js (hunk 2 above)
Comment (maartenhemmes) [reply src/data/customer/Setting.js]: @vdanylov try to use the same structure as we do on other functions as much as possible. Is there a particular reason you created this return structure?

File: src/data/customer/Setting.js (hunk 3 above)
Comment (vdanylov) [reply src/data/customer/Setting.js]: @maartenhemmes version changed to 0.1.99

**Pull request merged**